### PR TITLE
Css tuning

### PR DESCRIPTION
**Pull request by SamOlivas**
Various UI/UX changes.
* user profile
* detailed product
* reviews

### CODE REVIEW

File: src/components/Checkout.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { Elements, StripeProvider } from 'react-stripe-elements';
 import { connect } from 'react-redux';
 
-import StripCard from './StripeCard';
+import StripeCard from './StripeCard';
 import { getCart } from '../storeReducers/cartReducer';
```
Comment (nicknapoli82) [inline src/components/Checkout.js:7]: Got me on spelling!

File: src/components/DetailedProduct.js [hunk 2]
```diff
@@ -55,33 +55,39 @@ class DetailProduct extends React.Component {
...
+            <div>
+              Available:{' '}
+              {detailProduct.stock}
+            </div>
+            </div>
+          <div id='reviewProduct'>
+            <footer>
+              <h4>Reviews for the {detailProduct.name}</h4>
+              <Reviews product={detailProduct} productId={matchId} />
+            </footer>
```
Comment (nicknapoli82) [inline src/components/DetailedProduct.js:87]: I think HTML semantics suggest that a footer actually goes at the bottom of the document, but I think this is ok

**Review by nicknapoli82 — APPROVED** [review]
I think this is good work. Thanks @SamOlivas! Really appreciated. 
I don't see anything that changes any application logic, so I don't see a reason not to go ahead with merging this PR.

**Comment by SamOlivas** [comment]
Awesome!
Yeah I was trying to only touch things that would affect display. The organization of the css is definitely messy/unwieldy, but hey, at least it looks decent.

**Pull request merged**